src/Chirp.Core/DBFacade.cs

```diff
@@ -11,9 +11,8 @@ public DBFacade(IDatabase database)
         db = database;
     }
 
-    
 
-    public IEnumerable<Cheep> GetCheeps(string? authorUsername = null, int? limit = null)
+    public IEnumerable<Cheep> GetCheeps(int page, string? authorUsername = null)
```

Comment:
Do we not want to allow to get everything at once, or is this on purpose?
(I do not mind either way as long as its a conscious decision)

Reply:
It was on purpose but i guess we can always change it later if we for some reason need a full on timeline with all chirps